packages/components/tests/dummy/app/templates/components/stepper.hbs

```diff
@@ -6,6 +6,8 @@
   <p class="dummy-paragraph">A stepper indictor helps the user maintain context and directionality when advancing
     through a multi-step flow or feature, and in certain circumstances acts as a navigational device between steps. It
     is generally assembled as part of a larger stepper pattern.</p>
+  <p>For practical accesibility purposes, consider a stepper (the whole component) to be a list of items, with each step
+    being a list item.</p>
   <p class="dummy-paragraph">
     As of this writing the HDS team publishes two separate (but related)
     <code class="dummy-code">Stepper::Indicator</code>
```

Comment:
Below this line (on L14) `hierarchal` → `hierarchical`

Reply:
thanks for catching that, fixed! :thumbsup: